test/helpers/plugin_server.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	portInt, err := strconv.Atoi(port)
 	loggerx.ExitOnError(err, "while starting server")
 
-	binDir := filepath.Join(dir, "plugin-dist")
+	binDir := filepath.Join(dir, "../plugin-dist")
```

Comment:
What is the reason if this change?

Reply:
tests no have own `go.mod` you need to run server from `tests` folder and it breaks the flow as the `dir` is taken from the current path 🙂